src/univariate/discrete/dirac.jl

```diff
@@ -14,17 +14,17 @@ end
 
 eltype(::Dirac{T}) where {T} = T
 rand(rng::AbstractRNG, d::Dirac) = d.value
-pdf(d::Dirac, x::Real) = x == d.value ? 1 : 0
-cdf(d::Dirac, x::Real) = x < d.value ? 0 : 1
+pdf(d::Dirac, x::Real) = x == d.value ? 1.0 : 0.0
+cdf(d::Dirac, x::Real) = x < d.value ? 0.0 : 1.0
 quantile(d::Dirac{T}, p) where {T} = 0 <= p <= 1 ? d.value : T(NaN)
```

Comment:
This is personal opinion, but I like to use the unicode characters for math symbols (i.e. `≤` in this case).

Reply:
both are fine really

Reply:
err.... shouldn't it be `pdf(d::Dirac, x::Real) = x == d.value ? Inf : 0.0` ? (or maybe just not defining it for the Dirac delta)